src/pages/Ranking/RankingRecap/NextMonthSituation.tsx

```diff
@@ -5,12 +5,16 @@ import starIcon from "src/pages/Ranking/assets/star.svg";
 import graphIcon from "src/pages/Ranking/assets/graphIcon.svg";
 import styled from "styled-components";
 import localizedUrl from "src/utils/localizedUrl";
+import { ProgressRanking } from "./ProgressRanking/ProgressRanking";
 
 const StyledRecap = styled.div`
   border-top: 1px solid ${(p) => p.theme.colors.gray200};
   margin-top: ${(p) => p.theme.grid.sizes["4"]};
   padding-top: ${(p) => p.theme.grid.sizes["3"]};
   text-align: center;
+  .progress-container {
+    margin-bottom: ${(p) => p.theme.grid.sizes["4"]};
+  }
```

Comment:
questa non è una cosa da ProgressRanking? puoi spostarla là?

Reply:
Secondo me è corretto qui perché è il margine del componente in quel contesto (rispetto a ciò che c'è sotto o sopra). Il componente messo da un altra parte potrebbe aver bisogno di un margine diverso.

Reply:
ci sta, è solo poco comprensibile perché sta in mezzo alle cose del blocco generale, forse un componente <ProgressContainer> renderebbe il ragionamento più leggibile

Reply:
Ok fatto